README.md

```diff
@@ -23,7 +23,7 @@ options object.
   The maximum amount of time (seconds) the Reader will backoff for any single backoff
   event.
 * ```maxAttempts: 0``` <br/>
-  The number of times to a message can be requeued before it will be handed to the DISCARD handler and then automatically finished. 0 means that there is **no limit.** If no DISCARD handler is specified and `maxAttempts > 0`, then the message will be finished automatically when the number of attempts has been exhausted.
+  The number of times a given message will be attempted (given to MESSAGE handler) before it will be handed to the DISCARD handler and then automatically finished. 0 means that there is **no limit.** If no DISCARD handler is specified and `maxAttempts > 0`, then the message will be finished automatically when the number of attempts has been exhausted.
```

Comment:
For the `given to MESSAGE handler` do you mean the handler for `Reader.MESSAGE`?
I believe when the attempts of message reaches the `maxAttempts` the message will not be given to MESSAGE handler but directly thrown to `DISCARD` handler.(i.e times given to reader MESSAGE handler doesn't equal to `msg.attempts`)
https://github.com/dudleycarr/nsqjs/blob/master/src/reader.coffee#L150-L156